src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -471,8 +471,8 @@ public void SetRigiditySliderTooltip(float rigidity)
         var healthChangeLabel = GetNode<Label>(RigiditySliderTooltipHealthLabelPath);
         var mobilityChangeLabel = GetNode<Label>(RigiditySliderTooltipSpeedLabelPath);
 
-        float healthChange = rigidity * Constants.MEMBRANE_RIGIDITY_HITPOINTS_MODIFIER;
-        float mobilityChange = -1 * rigidity * Constants.MEMBRANE_RIGIDITY_MOBILITY_MODIFIER;
+        float healthChange = (float)Math.Round(rigidity * Constants.MEMBRANE_RIGIDITY_HITPOINTS_MODIFIER, 2);
```

Comment:
Rounding should be done when converting to string, for example `number.ToString("F3", CultureInfo.CurrentCulture)`
I think that works